internal/pkg/service/stream/api/service/source.go

```diff
@@ -298,6 +298,24 @@ func (s *service) EnableSource(ctx context.Context, d dependencies.SourceRequest
 	return s.mapper.NewTaskResponse(t)
 }
 
+func (s *service) ListSourceVersions(ctx context.Context, scope dependencies.SourceRequestScope, payload *api.ListSourceVersionsPayload) (res *api.EntityVersions, err error) {
+	if err := s.sourceMustExists(ctx, scope.SourceKey()); err != nil {
+		return nil, err
+	}
+
+	var versions []definition.Version
+	s.definition.Source().ListVersions(scope.SourceKey()).ForEach(func(value definition.Source, header *iterator.Header) error {
+		versions = append(versions, value.Version)
+		return nil
+	}).Do(ctx)
+
+	out := &api.EntityVersions{
+		Versions: s.mapper.NewVersionsResponse(versions),
+	}
+
+	return out, nil
```

Comment:
```suggestion
	return  &api.EntityVersions{
		Versions: s.mapper.NewVersionsResponse(versions),
	}, nil
```
If we do not work with this variable, return it right away.

Reply:
I wanted to avoid a discussion with @jachym-tousek-keboola as in the previous [PR](https://github.com/keboola/keboola-as-code/pull/2033) 😄